sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/ingestion/models/one/json/CommonSchemaEventLogFactory.java

```diff
@@ -0,0 +1,49 @@
+package com.microsoft.appcenter.analytics.ingestion.models.one.json;
+
+import com.microsoft.appcenter.analytics.ingestion.models.EventLog;
+import com.microsoft.appcenter.analytics.ingestion.models.one.CommonSchemaEventLog;
+import com.microsoft.appcenter.ingestion.models.Log;
+import com.microsoft.appcenter.ingestion.models.json.LogFactory;
+import com.microsoft.appcenter.ingestion.models.one.CommonSchemaLog;
+import com.microsoft.appcenter.ingestion.models.one.PartAUtils;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.LinkedList;
+
+/**
+ * Conversion of analytics logs to Common Schema.
+ */
+public class CommonSchemaEventLogFactory implements LogFactory {
+
+    @Override
+    public CommonSchemaEventLog create() {
+        return new CommonSchemaEventLog();
+    }
+
+    @Override
+    public Collection<CommonSchemaLog> toCommonSchemaLogs(Log log) {
+
+        /* Convert events only for now. */
+        if (log instanceof EventLog) {
+
+            /* 1 App Center log can map to multiple common schema logs if multiple keys. */
+            Collection<CommonSchemaLog> commonSchemaLogs = new LinkedList<>();
```

Comment:
In general yes CS logs can have multiple keys but at this stage we know the log is an EventLog and it will only contain one token so should we still go with the multiple keys approach for this log type?

Reply:
Not a problem since it's already enforced by Analytics just wanted to highlight it.

Reply:
This code will be likely shared with session log.